python/tvm/relay/query_layout.py

```diff
@@ -0,0 +1,20 @@
+

```

Comment:
We need to add the license header for all the new files introduce to the repo.

Reply:
fixed.